sys/include/net/gnrc/ipv6/nib/nc.h

```diff
@@ -123,9 +123,179 @@ extern "C" {
 /**
  * @brief   address is registered
  */
-#define GNRC_IPV6_NIB_NC_INFO_AR_STATE_REGISTERED       (0x0600)
+#define GNRC_IPV6_NIB_NC_INFO_AR_STATE_REGISTERED       (0x0400)
```

Comment:
out of curiosity: what's the reasoning for this change? Cosmetics, or is there a more sophisticated motive behind this?